app/assets/javascripts/application.js

```diff
@@ -0,0 +1,46 @@
+$( function() {
```

Comment:
are these two files, application.js and application.css are going to be included in all pages, including the public, non-admin pages? if so, better to only include them into our new CMS layout